README.rst

```diff
@@ -140,6 +173,7 @@ add a map annotation corresponding to the configuration above:
 Now restart OMERO.web as normal for the configuration above to take effect.
 You should now be able to browse to a ``Genes`` page and search for
 ``CDC20`` or ``ENSG00000117399``.
+Note that if you add another set of Map Annotations with the same Namespace they will not be added to the existing set but will get a new "paragraph" with the label again. It is therefore advisable for the sake of visible clarity to copy the Key-Value pairs, remove the old set, add the KV-pairs to the new set and then create a new Map Annotation with the combined KV-pairs.
```

Comment:
I'm not sure about the advice of keeping everything in a single Map Annotation...
Mapr generally assumes that the Key-Value pairs for each config (e.g. Gene) will be in a Map-Annotation with it's own "..gene" namespace. So that is certainly the best tested workflow. However, I'm not too sure what functionality of Mapr will be affected if you combine all Key-Value pairs into a single Map Annotation.
Certainly if you don't use different namespaces then you won't get a custom `Label` on each Annotation (paragraph).
Looking at the code it appears that `namespace` is also used in the search query: see https://github.com/ome/omero-mapr/blob/bd2bcf6eb281927075094b2b9e460d856f314815/omero_mapr/tree.py#L96
but I haven't tested. If so, then I would certainly advise that Key-Value pairs be kept in separate Map Annotations with the correct namespace.

Reply:
I think you misunderstand me.
My intention is not to put all KV-pairs of an image into the same MapAnnotation with one Namespace resulting in one large"paragraph".
I was simply trying to warn, that if one creates a new MapAnnotation with a Namespace that already exists it will show up as a new paragraph, resulting in two different "paragraphs" with the same label (Namespace). Which takes up a lot of unnecessary space. 
The nicer solution would be to check for existing MapAnnotations with that Namespace one wants to use, get the KV-pairs of the "old" one and append them to the "new" KV-pairs and finally create __one__ new MapAnnotation with that namespace resulting in __one__ slightly longer paragraph with the label for that Namespace instead of two paragraphs where you always have the large header consisting of label and "created by..."

Reply:
Do you have a proposal how to phrase that better in the Readme, so it is more understandable?

Reply:
Sorry, yes - I misunderstood as I was reading that addition out of context (and not reading carefully)!
Reading again, your text is actually fine, but here's an alternative if you want (happy to go with either):

Note that if you create multiple Map Annotations with the same Namespace, this will result in a more verbose layout in the webclient UI. It is preferable to combine all Key-Value pairs for a particular Namespace into a single Map Annotation.

Reply:
Great. 
Thanks for the suggestion, I implemented it.
Things just sound better most of the time coming from a native speaker 👍